src/acl/CharacterSetOption.h

```diff
@@ -29,10 +29,10 @@ TypedOption<CharacterSetOptionValue>::import(const SBuf &rawValue) const
 }
 
 template <>
-inline
-void
-TypedOption<CharacterSetOptionValue>::print(std::ostream &os) const
+inline void
+TypedOption<CharacterSetOptionValue>::printValue(std::ostream &os) const
 {
+    os << '=';
```

Comment:
This part belongs to the common TypedOption::print() code AFAICT -- all configured() and valued() options should print `=` before the value; that symbol is a part of ACL option value syntax.